src/i9n/runtime.rs

```diff
@@ -35,16 +36,22 @@ impl Runtime {
         // Start rgbd on a separate thread
         if config.threaded {
             let rgbd_opts = rgbd::Opts {
-                verbose: 5,
                 bin_dir: String::new(),
                 data_dir: config.data_dir.clone(),
                 contracts: config
                     .contract_endpoints
                     .iter()
                     .map(|(k, _)| k.clone())
                     .collect(),
+                fungible_rpc_endpoint: config
+                    .contract_endpoints
+                    .get(&ContractName::Fungible)
+                    .unwrap()
```

Comment:
Generally we avoid `unwraps` as much as possible, b/c each of them will mean the whole app crashing (like mobile app) without any notice or explanation

Reply:
fixed it with an `unwrap_or` containing default value.

I tried replacing `unwrap` with `?`, but the error is returned there is a `std::option::NoneError` (which we cannot add to the `BootstrapError` without adding `#![feature(try_trait)]`, see <https://github.com/rust-lang/rust/issues/42327> for more information)

Reply:
You need to use `.ok_or(Error::ErrorType)` for optionals (where `ErrorType` may be a new or existing error type, feel free to add one if needed). Using other endpoint does not make sense in this context and will make app behavior unpredictable/unexplainable.

Reply:
thanks for the tip, just changed it